pgoapi/pgoapi.py

```diff
@@ -393,6 +397,36 @@ def cleanup_pokemon(self, inventory_items=None):
                             self.log.info("Failed to release Pokemon %s", pokemon)
                         sleep(3)
 
+        if self.RELEASE_DUPLICATES:
+            for pokemons in caught_pokemon.values():
+                if len(pokemons) > MIN_SIMILAR_POKEMON:
```

Comment:
we can try to run this code in the same loop at the one above it. can you make it a part of that?